databuilder/extractor/snowflake_metadata_extractor.py

```diff
@@ -4,17 +4,15 @@
 
 import logging
 from collections import namedtuple
-
-from pyhocon import ConfigFactory, ConfigTree
+from itertools import groupby
 from typing import Iterator, Union, Dict, Any
-from unidecode import unidecode
 
 from databuilder import Scoped
 from databuilder.extractor.base_extractor import Extractor
 from databuilder.extractor.sql_alchemy_extractor import SQLAlchemyExtractor
 from databuilder.models.table_metadata import TableMetadata, ColumnMetadata
-from itertools import groupby
-
+from pyhocon import ConfigFactory, ConfigTree
```

Comment:
typical python lib order based on https://www.python.org/dev/peps/pep-0008/
```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
```

Even we don't have CI to enforce it, it would good to still use this model (e.g databuilder import should come last) WDYT?

Reply:
@feng-tao Do you mind if I setup a CI to check import order?

Reply:
that will be great! thanks @dungdm93

Reply:
`isort_check` [added](https://github.com/amundsen-io/amundsendatabuilder/pull/428/commits/a45f7cf971411d39ed1a739c88259f61e4614cae).
I also add `.editorconfig` file to make the code more consistent cross IDE.

@feng-tao could you please take a look.